examples/window-app/silabs/src/WindowManager.cpp

```diff
@@ -56,6 +56,12 @@ using namespace ::chip::DeviceLayer::Silabs;
 #define APP_STATE_LED 0
 #define APP_ACTION_LED 1
 
+#ifdef DIC_ENABLE
+#define DECIMAL 10
+#define BYTE 5
```

Comment:
This naming is odd, BYTE doesn't convey that it is the size of your buffer nor why 5 is used